### PR TITLE
WA groups: enable the feature flag

### PR DESCRIPTION
**Pull request by AmishaBisht**
## Summary
 Target issue is #3380 
create the feature flag for whatsapp group feature

### CODE REVIEW

File: test/glific_web/schema/organization_test.exs [hunk 1]
```diff
@@ -455,6 +455,7 @@ defmodule GlificWeb.Schema.OrganizationTest do
     assert services["flow_uuid_display"] == false
     assert services["contact_profile_enabled"] == false
     assert services["auto_translation_enabled"] == false
+    assert services["whatsapp_group_enabled"] == false
```
Comment (AkhileshNegi) [inline test/glific_web/schema/organization_test.exs:458]: how come you have not updated this file
`assets/gql/organizations/get_services.gql`
and still getting false value

File: lib/glific/misc/flags.ex [hunk 2]
```diff
@@ -198,6 +198,25 @@ defmodule Glific.Flags do
...
+        true
+
+      trusted_env?(app_env, organization.id) ->
+        true
+
+      true ->
+        false
+    end
+  end
+
```
Comment (AkhileshNegi) [inline lib/glific/misc/flags.ex:219]: this is starting to get a repeated code see if u can optimise this and use single function

**Review by AkhileshNegi — CHANGES_REQUESTED** [review]

File: test/glific_web/schema/organization_test.exs (hunk 1 above)
Comment (AmishaBisht) [reply test/glific_web/schema/organization_test.exs:458]: it is there

File: lib/glific/misc/flags.ex (hunk 2 above)
Comment (AmishaBisht) [reply lib/glific/misc/flags.ex:219]: okay sure

**Review by AkhileshNegi — APPROVED** [review]

**Pull request merged**